activities/philanthropic-fundraising/King-Baudouin.md

```diff
@@ -0,0 +1,16 @@
+---
+type: Guide
+---
+
+# How to donate through the King Baudouin Foundation
+
+The Foundation for Public Code has set up a so called "Advised Gift Fund" at the King Baudouin Foundation (KBFUS) in the U.S. 
```

Comment:
"So called" could be read as sarcastic. I've put 'advised gift fund' in lower case because it's a generic term - it doesn't have to be a specific quotation.
```suggestion
The Foundation for Public Code has set up an advised gift fund at the King Baudouin Foundation US (KBFUS) in the U.S. 
```

Reply:
KBFUS wrote in with capitals, so followed that example but will apply your suggestions. Thanks for pointing out "so called" good to know

Reply:
Capitalizing jargon highlights how special it is, and makes it seem more impressive and unapproachable. Writing generic terms with lower case makes them symobolically accessible.